base/ubi9/entrypoint.sh

```diff
@@ -5,6 +5,16 @@ if [ ! -d "${HOME}" ]; then
   mkdir -p "${HOME}"
 fi
 
+# Configure container builds to use vfs or fuse-overlayfs
+if [ ! -d "${HOME}/.config/containers" ]; then
```

Comment:
@dkwon17 with this change we should probably update the docs as well, right?
e.g. https://docs.redhat.com/en/documentation/red_hat_openshift_dev_spaces/3.17/html-single/user_guide/index?extIdCarryOver=true&sc_cid=701f2000001Css5AAC#enabling-overlay-with-a-configmap
Now UDI will be smart enough to detect fuse automatically

Reply:
@ibuziuk , yes, a configmap will not be needed anymore if the tooling container image  is based on ubi/udi